gcloud/storage/test_blob.py

```diff
@@ -990,22 +990,36 @@ def test_storage_class(self):
         self.assertEqual(blob.storage_class, STORAGE_CLASS)
 
     def test_time_deleted(self):
+        import datetime
         BLOB_NAME = 'blob-name'
         connection = _Connection()
         bucket = _Bucket(connection)
-        TIME_DELETED = '2014-11-05T20:34:37Z'
+        TIME_DELETED = '2014-11-05T20:34:37.000Z'
+        TIMESTAMP = datetime.datetime(2014, 11, 5, 20, 34, 37)
```

Comment:
Should this one be computed via `strptime` from `TIME_DELETED`?  I guess they won't likely drift since they are right together.